aips/aip-116.md

```diff
@@ -234,17 +241,50 @@ message-resources = \\*( LF "- " URI )
 
 ### Domain Separated Signing Algorithm
 
-A plain text construction of a SIWA message must be prefixed with a `SIGN_IN_WITH_APTOS::` hash before signing to ensure domain separation across other signatures. The SIWA signing messages must conform to the following expression:
+A plain text construction of a SIWA message (as described [here](#siwa-message-construction)) must be prefixed with a `SIGN_IN_WITH_APTOS::` hash before signing to ensure domain separation across other signatures. The SIWA signing messages must conform to the following expression:
 
 ```
 sha3_256(b"SIGN_IN_WITH_APTOS::") || <siwa_message>
 ```
 
 where `<siwa_message>` is the plain text construction of a SIWA message explained in the [Message Construction](#message-construction) section.
 
+### Signature and Message Verification
```

Comment:
Can we have a proper term for this backend SIWA verification? Maybe we can call it "**Backend SIWA verification**"?